core/db_models/EEM_Form_Element.model.php

```diff
@@ -148,16 +149,23 @@ protected function __construct(FormStatus $form_status, InputTypes $input_types,
                         sprintf(
                             /* translators: 1 class name */
                             __(
-                                'Whether form input is active, archived, trashed, or used as a default on new forms. Values correspond to the %s constants.',
+                                'Whether form element is active, archived, trashed, or used as a default on new forms. Values correspond to the %s constants.',
                                 'event_espresso'
                             ),
-                            'EEM_Form_Element::STATUS_*'
+                            'EventEspresso\core\services\form\meta\FormStatus'
```

Comment:
LOLZ!!!

VS Code rename needs work !

Reply:
oh those were wrong the way they were anyways... so that change is kinda better in some ways 🙄

Reply:
oh duh... 🤦🏻‍♂️ 
your change is great... 
forgot those constants were renamed when I extracted them.

**nothing to see here! move along** :disappointed: